pkg/apis/compliance/v1alpha1/tailoredprofile_types.go

```diff
@@ -6,6 +6,21 @@ import (
 
 // FIXME: move name/rationale to a common struct with an interface?
 
+// DisableOutdatedReferenceValidation a label is used to disable validation of outdated references
+const DisableOutdatedReferenceValidation = "compliance.openshift.io/outdated-reference-validation-disable"
+
+// PurneOutdatedReferencesAnnotationKey is the annotation key used to indicate that the outdated references of rules or variables should be pruned
+const PurneOutdatedReferencesAnnotationKey = "compliance.openshift.io/prune-outdated-references"
+
+// RuleLastCheckTypeChangedAnnotationKey is the annotation key used to indicate that the rule check type has changed, store its previous check type
+const RuleLastCheckTypeChangedAnnotationKey = "compliance.openshift.io/rule-last-check-type"
+
+// VariableDeprecatedAnnotationKey is the annotation key used to indicate that the variable is deprecated
+const VariableDeprecatedAnnotationKey = "compliance.openshift.io/variable-deprecated"
```

Comment:
What if we had `compliance.openshift.io/deprecated`? Would we be able to reuse the same annotation for both variables and rules?